renderer/screens/oracles/machines.js

```diff
@@ -470,6 +470,7 @@ export const createNewVotingMachine = (epoch, address) =>
                     feePerGas,
                     minOracleReward,
                     oracleReward: minOracleReward,
+                    totalVotingFunds: minOracleReward * context.committeeSize,
```

Comment:
not sure we need this as it's derived from other params === dynamically calculated

Reply:
or, leave this calculation here in the machine, and just bind it to the form field.

looks like even better approach